tests/conftest.py

```diff
@@ -62,7 +62,7 @@ def testfunc():
             # Call actual test function
             return pyfuncitem.obj()
         except Exception:
-            bucket[0] = sys.exc_info
+            bucket[0] = sys.exc_info()
```

Comment:
Just a minor suggestion: `bucket = [None]` and `bucket[0] = sys.exc_info()` could be replaced with `exc_info = None`, `nonlocal exc_info` in the closure definition, and `exc_info = sys.exc_info()` in the same place.

Reply:
reads less weird! thx